plugin/src/main/java/io/jenkins/plugins/analysis/warnings/Valgrind.java

```diff
@@ -0,0 +1,41 @@
+package io.jenkins.plugins.analysis.warnings;
+
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.jenkinsci.Symbol;
+import hudson.Extension;
+
+import io.jenkins.plugins.analysis.core.model.AnalysisModelParser;
+import io.jenkins.plugins.analysis.core.model.StaticAnalysisLabelProvider;
+import io.jenkins.plugins.analysis.core.model.SymbolIconLabelProvider;
+
+/**
+ * Provides a parser and customized messages for Valgrind.
+ *
+ * @author Michael Trimarchi
+ */
+public class Valgrind extends AnalysisModelParser {
+    private static final long serialVersionUID = 1L;
+    private static final String ID = "valgrind";
+
+    /** Creates a new instance of {@link Valgrind}. */
+    @DataBoundConstructor
+    public Valgrind() {
+        super();
+        // empty constructor required for stapler
+    }
+
+    /** Descriptor for this static analysis tool. */
+    @Symbol("Valgrind")
```

Comment:
Needs to be lowercase:

```suggestion
    @Symbol("valgrind")
```